src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  justify-content: space-between;
+  gap: 46px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
```

Comment:
The media queries should be ordered from smallest to largest. The media query for min-width: 488px should be first, then min-width: 768px, then min-width: 1024px, and finally min-width: 1200px.